packages/website/src/components/RecruitmentModal/RecruitmentModal.tsx

```diff
@@ -36,8 +40,16 @@ export const RecruitmentModal = ({ isOpen, config, onClose, modalType }: Recruit
         boxShadow="large"
         textAlign="center"
       >
+        <IconButton
```

Comment:
🤔 Closing icon is the first element that gets focused when you open the modal. I'm no sure if it should work that way. Maybe it would be better if we put autofocus to the name input.